src/assets/i18n/en.json

```diff
@@ -31,6 +31,8 @@
     "SELECT_GEO_HAZARD": "Select geo hazard",
     "VIEW_WARNINGS": "View warnings from varsom.no",
     "TOGGLE_OBSERVATION": "Switch between simple and full observation",
+    "TOGGLE_OBSERVATION_EXTRA_INFO_1": "Toggle on to send observations at your position - simple and fast. This is extremely useful for the warning services and for other users.",
```

Comment:
Helst ikke bruk *_1 og *_2, det blir vanskelig for de som skal oversette å forstå.
Styr heller linjeskift i oversettelses-teksten, hvis det er grunnen til at disse er skilt.

Reply:
fant en løsning med \n \n og white-space: pre-line